app/helpers/hashPassword.js

```diff
@@ -0,0 +1,8 @@
+const bcrypt = require('bcrypt');
+
+const hashPassword = async password => {
```

Comment:
A good way of thinking about helper functions is that they should be as generic as posible, to make them more reusable. So instead of making a `hashPassword` function it's preferable to have a `hashSomething` or `hashString` function. What do you think?
P.D.: The same goes to the file name